hosted-tenant-base/pom.xml

```diff
@@ -92,6 +92,13 @@
                 <version>${vespaversion}</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.junit.jupiter</groupId>
+                <artifactId>junit-jupiter-api</artifactId>
+                <version>${vespa.junit.version}</version>
+                <scope>provided</scope>
```

Comment:
This should be test scope right?

Reply:
Oops, yes. But then I wonder why those applications were failing ...